consensus/raft/logging.go

```diff
@@ -58,19 +58,38 @@ func (fw *logForwarder) Write(p []byte) (n int, e error) {
 }
 
 func (fw *logForwarder) repeated(t int, msg string) bool {
-	if fw.last == nil {
-		fw.last = make(map[int]*lastMsg)
+	if fw.lastMsgs == nil {
+		fw.lastMsgs = make(map[int][]string)
+		fw.lastTip = make(map[int]time.Time)
+	}
+
+	// We we haven't tipped about repeated log messages
```

Comment:
small typo "We we"

Reply:
also I think it should be "in a while" not "in while" if I understand the context correctly